### PR TITLE
Add June Conference Events to News

### PR DESCRIPTION
**Pull request by garyanewsome**
###### Add June Conference Events to News Page

Added a news article showing June conference events Cornerstone will be attending.

###### Needs
- code review
- approval
- deployed

### CODE REVIEW

File: source/blog/2017-05-10-Cornerstone-this-June.html.erb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+</a> 
+
+<br>
+<br>
+
+<p>As well as the,<br>
+<strong>Medical Affairs Professional Society (MAPS) 2017 Annual Meeting.</strong><br>
+in National Harbr, MD, from June 20-22, 2017.</p>
+<br>
+<a href="http://www.medicalaffairs.org/annual-meeting/">
```
Comment (ninjapanzer) [inline source/blog/2017-05-10-Cornerstone-this-June.html.erb]: Should like to Speakers page http://www.medicalaffairs.org/annual-meeting/speakers/ But their site is borked 🖕

File: source/blog/2017-05-10-Cornerstone-this-June.html.erb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+author: Cornerstone Management Resourse Systems
+location: CARNEGIE, Pa.
+category: News
+tags: Cornerstone, Carl Thorensen, Joe Thorensen, Kathy Ireland
+---
+<p>This June Cornerstone is proud to be participating in several events:</p>
+<p><strong>The Life Science Trainers and Educators Network's (LTEN) 46th Annual Conference.</strong><br>
+in Nashville, TN, from June 5-8, 2017.</p>
+<br>
+<a href="http://www.l-ten.org/">
```
Comment (ninjapanzer) [inline source/blog/2017-05-10-Cornerstone-this-June.html.erb]: http://www.l-ten.org/members/?id=12131636 🖕 to ltenconference

**Review by ninjapanzer — CHANGES_REQUESTED** [review]

File: source/blog/2017-05-10-Cornerstone-this-June.html.erb (hunk 1 above)
Comment (garyanewsome) [reply source/blog/2017-05-10-Cornerstone-this-June.html.erb]: Going to leave this link blank, and when we get more info on MAPS we will redirect it to our MAPS landing page

Comment (ninjapanzer) [reply source/blog/2017-05-10-Cornerstone-this-June.html.erb]: roger

Comment (garyanewsome) [reply source/blog/2017-05-10-Cornerstone-this-June.html.erb]: wilko

**Review by ninjapanzer — APPROVED** [review]

**Pull request merged**